toml/src/main/resources/META-INF/toml-only.xml

```diff
@@ -32,6 +32,11 @@
                          enabledByDefault="true" level="WARNING"
                          implementationClass="org.rust.toml.inspections.CrateNotFoundInspection"/>
 
+        <intentionAction>
+            <className>org.rust.toml.intentions.ExpandDependencySpecificationIntention</className>
+            <category>Cargo.toml</category>
```

Comment:
```suggestion
            <category>Rust/Cargo.toml</category>
```
`<category>Cargo.toml</category>` creates a new category on the top level but this intention should belong to `Rust` instead